apps/etterlatte-saksbehandling-ui/client/src/components/person/journalfoeringsoppgave/VelgJournalpost.tsx

```diff
@@ -69,20 +72,55 @@ export default function VelgJournalpost({ journalpostId }: { journalpostId: stri
     <>
       {isPending(journalposter) && <Spinner label="Henter journalposter for bruker" visible />}
       {isPending(journalpostStatus) && <Spinner label="Henter journalpost for bruker" visible />}
+      {isFailure(journalpostStatus) && (
+        <ApiErrorAlert>Feil ved henting av journalpost. Kan ikke fortsette behandlingen.</ApiErrorAlert>
+      )}
 
       {journalpost ? (
         <>
-          <Heading size="medium">{journalpost.tittel}</Heading>
-          <Tag variant="alt1">{journalpost.avsenderMottaker?.navn}</Tag>
+          <Panel>
+            <Heading size="medium" spacing>
+              Journalpost ({journalpost.journalpostId})<Detail>{journalpost.tittel}</Detail>
+            </Heading>
+            <InfoWrapper>
+              <Info
+                label="Avsender/mottaker"
+                tekst={
+                  <>
+                    {journalpost.avsenderMottaker?.navn} (
+                    {journalpost.avsenderMottaker?.id?.length === 11 ? (
+                      <Link href={`/person/${journalpost.avsenderMottaker?.id}`} target="_blank">
```

Comment:
er det vits med noopener/noreferer?

Reply:
Kanskje ikke. La det til for å sikre at sb ikke blir "kastet ut" av oppgaven, men kanskje bedre å la de ta stilling til det selv.

Reply:
Antok du mente `target` siden `target` og `rel` (noopener noreferrer) er to forskjellige ting og sistnevnte ikke er brukt her.